docs/plugin_dev.rst

```diff
@@ -0,0 +1,299 @@
+            use the classmethod load.
+            """
+
+        @classmethod
+        def depends_on(cls) -> List[str]:
+            """
+            This returns a list of modules that your plugin needs. This can be another plugin module
+            or one of the built-in modules, which are listed in SkyTemple's setup.py
+            (or in the future it's pyproject.toml).
+
+            You can reference these other modules and rely on functionality in them.
+            """
+            return []
+
+        @classmethod
+        def sort_order(cls) -> int:
+            """
+            A number that is used to sort all of the items in the main item tree of the SkyTemple UI.
+
+            Experiment with this until you find a value you are happy with.
+            """
+            return 0
+
+        def load_tree_items(self, item_tree: ItemTree):
+            """
+            This is the heart of your plugin (if your plugin's purpose is to show views in the UI.
+            You can add new views to the main item tree on the left of SkyTemple's UI here.
+
+            You must implement this, but you can also do just nothing,
+            if your UI does not actually provide new views.
+
+            You can also manipulate other items in the item tree, but this is not recommended, since
+            it could easily break with updates.
+            """
+            pass
+
+.. _GPLv3+: https://www.gnu.org/licenses/gpl-3.0.html.en
+.. _pyproject.toml: https://setuptools.pypa.io/en/latest/userguide/pyproject_config.html
+
+Dependencies
+------------
+Your plugin can not pull in any dependencies not bundled with SkyTemple, if you intend
+to distribute it via the ``plugins`` directory (see "`Via SkyTemple's plugin loader`_").
+
+You can rely on the following dependencies being bundled with SkyTemple.
+
+SkyTemple 1.6
+~~~~~~~~~~~~~
+- GTK 3.24 (Note: SkyTemple will eventually switch to GTK 4. Try to only use GTK 4 compatible widgets).
+- ndspy 3
+- skytemple-files 1.6
+- pmdsky-debug-py in the same version skytemple-files requires
+- skytemple-dtef 1.6
+- skytemple-icons 1.3
+- explorerscript 0.1
+- skytemple-rust 1.6
+- dungeon_eos 0.0.5+
+- range-typed-integers 1
+- pygobject 3.40.0+
+- pycairo 1.18+
+- tilequant 1
+- skytemple-ssb-debugger 1.6
+- CairoSVG 2.7
+- packaging
+- wheel
+- importlib_metadata on Python < 3.10
+- importlib_resources on Python < 3.9
+
+
+Example Plugin
+--------------
+An example plugin that also shows a lot of the APIs you can use and how to build custom views
+can be found at:
+
+https://github.com/SkyTemple/skytemple-example-plugin/
+
+Additionally you can reference
+`built-in modules <https://github.com/SkyTemple/skytemple/tree/master/skytemple/module>`_.
+
+Distribution
+------------
+In order to distribute your plugin you need to create a Wheel for it.
+
+To do this, install the ``wheel`` package on your system via ``pip``.
```

Comment:
```suggestion
To do this, install the ``wheel`` package on your system with ``pip install wheel``.
```
You might as well add the command here for people who aren't familiar with installing pip packages (in my setup, it's `pip3`, but I suppose the binary name depends on the platform maybe?).